mlrun/runtimes/base.py

```diff
@@ -462,6 +463,8 @@ def run(
                 last_err = err
                 result = self._post_run(task=runspec, err=err)
 
+        self._clean_run(result, execution)  # hook for runtime specific cleanup
```

Comment:
Maybe `_post_run` and then can be used not only for cleanup

Reply:
we already have _post_run()

Reply:
done